int16 cid
int16 oid
float32[3] pos
time update_time
bool active